akka24-conductr-client-lib/src/main/scala/com/typesafe/conductr/clientlib/akka/BundlesConnector.scala

```diff
@@ -0,0 +1,298 @@
+        Source.actorPublisher(props(conductrAddress, getBundlesEventsRequest, getBundlesRequest, Some(stopAfter), expectingEventBurstTimeout, reconnectInterval))
+    }
+
+  /**
+   * Client API which provides a nicer interface to the [[BundlesConnector]] actor.
+   *
+   * Provides a means for the caller to connect to the ConductR Bundles endpoint continuously.
+   *
+   * Returns a flow graph of `Flow[(HttpRequest, HttpRequest), Seq[Bundle], NotUsed]`.
+   *
+   * The input is a tuple of [[HttpRequest]]:
+   * - The first element of the tuple is the HTTP request for ConductR Bundles Events endpoint.
+   * - The second element of the tuple is the HTTP request for ConductR Bundles endpoint.
+   *
+   * The `Seq[Bundle]` will be emitted whenever there's any change of the bundle state within ConductR.
+   * The change detection is derived from having Bundle SSE propagated by ConductR Bundles Events endpoint, and an
+   * updated `Seq[Bundle]` being recognized by the [[BundlesConnector]] actor.
+   *
+   * @param conductrAddress the ConductR Control Protocol base URL
+   * @param expectingEventBurstTimeout the expected amount of time of which SSE event burst from `/bundles/events` will
+   *                                   take place. Once event burst is over, the latest bundle state will be obtained
+   *                                   from `/bundles/events`
+   * @param reconnectInterval the amount of time to wait to reconnect to `/bundles/events` when disconnected.
+   * @param system the actor system which is used to build [[BundlesConnector]]
+   * @return the flow graph of `Flow[(HttpRequest, HttpRequest), Seq[Bundle], NotUsed]`.
+   */
+  def continuously(
+    conductrAddress: URL,
+    expectingEventBurstTimeout: FiniteDuration = 500.millis,
+    reconnectInterval: FiniteDuration = 500.millis
+  )(onBundleUpdates: Seq[Bundle] => Unit)(implicit system: ActorRefFactory): Flow[(HttpRequest, HttpRequest), Seq[Bundle], NotUsed] =
+    Flow[(HttpRequest, HttpRequest)].flatMapConcat {
+      case (getBundlesEventsRequest, getBundlesRequest) =>
+        Source.actorPublisher(props(conductrAddress, getBundlesEventsRequest, getBundlesRequest, None, expectingEventBurstTimeout, reconnectInterval))
+    }
+
+  private type Connection = Flow[HttpRequest, HttpResponse, Future[Http.OutgoingConnection]]
+
+  sealed trait Message
+
+  private case object ConsumeBundlesEvents extends Message
+  private case class ConsumeBundlesEventsOutcome(outcome: Try[Done]) extends Message
+
+  private case object GetBundles extends Message
+  private case class GetBundlesSuccess(bundles: Seq[Bundle]) extends Message
+  private case class GetBundlesError(error: Throwable) extends Message
+
+  sealed trait State
+
+  private object Initializing extends State
+  private object ExpectingEvent extends State
+  private object ExpectingEventBurst extends State
+
+  case object TimeoutException extends RuntimeException
+
+  private def props[T](
```

Comment:
This is private because I'd like to expose only `BundlesConnector.connect()` which creates the flow graph.

Also we don't have a `Name` declaration because the `BundlesConnector` actor will only be created through `Source.actorPublisher()` which accepts`BundlesConnector.props()`.